### PR TITLE
Initial check in for the sdk generation core project

### PR DESCRIPTION
**Pull request by raych1**
- Copy the existing code from openapi-alps repo
- Remove dependencies on the Rush and Gulp, use Npm and TSC as replacement
- Add lint

### CODE REVIEW

File: tools/sdk-automation-core/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+lib/
+node_modules/
```
Comment (weshaggard) [inline tools/sdk-automation-core/.gitignore:2]: We should eliminate tool specific gitignore and add things to the root one as needed.

File: tools/sdk-automation-core/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+    MIT License
```
Comment (weshaggard) [inline tools/sdk-automation-core/LICENSE:1]: This should be removed as we have it in the license.

File: tools/sdk-automation-core/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+
+# Contributing
```
Comment (weshaggard) [inline tools/sdk-automation-core/README.md:2]: This should be removed.

**Review by weshaggard — APPROVED** [review]
Added a few comments to clean-up as we move over but I'm fine with the initial source dump. I would like to get a pipeline up and running as soon as possible.

File: tools/sdk-automation-core/README.md (hunk 3 above)
Comment (raych1) [reply tools/sdk-automation-core/README.md]: Removed.

File: tools/sdk-automation-core/LICENSE (hunk 2 above)
Comment (raych1) [reply tools/sdk-automation-core/LICENSE]: Removed.

File: tools/sdk-automation-core/.gitignore (hunk 1 above)
Comment (raych1) [reply tools/sdk-automation-core/.gitignore]: Consolidated to the gitignore under the root folder.

**Pull request merged**